### PR TITLE
Move CommandFailedException to exceptions.py

### PR DESCRIPTION
**Pull request by RazTamir**

### CODE REVIEW

File: ocs/exceptions.py [hunk 1]
```diff
@@ -4,3 +4,7 @@ class CommandFailed(Exception):
 
 class UnsupportedOSType(Exception):
     pass
+
+
+class CommandFailedException(Exception):
```
Comment (vasukulkarni) [inline ocs/exceptions.py]: we have CommandFailed Exception in the same file, probably this was overlooked before during the merge, we can use the original one and delete the extra in install.py

Comment (RazTamir) [reply ocs/exceptions.py]: right,

Fixing now.
So now I just need to:
git commit (no --amend)
and push new commit?

Comment (clacroix12) [reply ocs/exceptions.py]: @RazTamir yeah you can just push a new commit and upon merging to master if you choose to squash your commits into one github will let you do that

**Comment by vasukulkarni** [comment]
@clacroix12 needs 2 approvals, please squash and merge

**Pull request merged**